CHANGELOG.md

```diff
@@ -3,9 +3,13 @@
 ### Features
 - Allow nullable `error_after` in source freshness ([#3874](https://github.com/dbt-labs/dbt-core/issues/3874), [#3955](https://github.com/dbt-labs/dbt-core/pull/3955))
 - Increase performance of graph subset selection ([#4135](https://github.com/dbt-labs/dbt-core/issues/4135),[#4155](https://github.com/dbt-labs/dbt-core/pull/4155))
+
 ### Fixes
 - Changes unit tests using `assertRaisesRegexp` to `assertRaisesRegex`
 
+### Under the hood
+Bump artifact schema versions for 1.0.0
```

Comment:
Could we include a brief overview of what structurally changed in `manifest`, `run-results`, + `sources`? For easy reference by users, and by me when I go to update docs.getdbt.com

Reply:
Oh sorry you asked for a brief description and not just what version was bumped...

Reply:
yeah! I have a sense it's very small things that changed

Reply:
There are more changes than would make sense to write down but I listed the big ones. The small ones are some places we changed `int` to `float` but I listed renames and bigger things like that